app/controllers/ems_common.rb

```diff
@@ -437,6 +437,28 @@ def button
         end
         return
       end
+      if params[:pressed] == "ems_cloud_recheck_auth_status"
+        @record = find_by_id_filtered(model, params[:id])
+        authentications = @record.authentication_for_summary
+        authentications.collect do |auth|
+          result, details = @record.authentication_check_queue(auth[:authtype], :save => true)
+          label =
+              case auth[:authtype]
+                when "default" then _("Default")
+                when "metrics" then _("C & U Database")
+                when "amqp"    then _("AMQP")
+                else;               _("<Unknown>")
```

Comment:
This case statement is already in 3 other places.  For example: https://github.com/ManageIQ/manageiq/blob/fa42171d9cac349eea580a96c134384de949a5d2/app/helpers/ems_infra_helper/textual_summary.rb#L152-L158

```
app/helpers/ems_cloud_helper/textual_summary.rb:        when "metrics" then _("C & U Database")
app/helpers/ems_infra_helper/textual_summary.rb:        when "metrics" then _("C & U Database")
app/helpers/ems_network_helper/textual_summary.rb:        when "metrics" then "C & U Database"
```

Maybe we can ask an authentication decorator for this friendly name.  cc @martinpovolny